### PR TITLE
feat(): add snapshot profile resolver

### PR DESCRIPTION
**Pull request by midgerate**

### CODE REVIEW

File: src/resolvers/snapshot.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      `
+        }
+      })
+    ).data.data.user;
+    if (!user) throw new Error('User not found');
+    if (!user.avatar) throw new Error('User avatar not set');
+
+    // TODO: how to get the avatar from our own ipfs function
+    // TODO: ipfs is naturally slow, so what should be the strategy as we wait for all the resolvers to resolve.
+    const ipfsUrl = `https://ipfs.io/ipfs/${user.avatar.replace('ipfs://', '')}`;
```
Comment (bonustrack) [inline src/resolvers/snapshot.ts:31]: This gateway has lot of limitations, I would recommend use cloudflare-ipfs.com instead

**Comment by midgerate** [comment]
@bonustrack please review again.

**Comment by bonustrack** [comment]
Also add Snapshot as resolver in readme

**Comment by midgerate** [comment]
@bonustrack done changes you asked for.

File: src/utils.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { createHash } from 'crypto';
 import sharp from 'sharp';
 import { StaticJsonRpcProvider } from '@ethersproject/providers';
+import * as snaphsotjs from '@snapshot-labs/snapshot.js';
```
Comment (bonustrack) [inline src/utils.ts:4]: snaphsotjs?
```suggestion
import snapshot from '@snapshot-labs/snapshot.js';
```

**Review by bonustrack — APPROVED** [review]
Looks good

**Pull request merged**